CONTRIBUTING.md

```diff
@@ -31,6 +25,33 @@ Some examples:
 - `make run` build the base image and run a bash shell in a container based in it in interactive mode.
 - `make run-local` volume the build scripts and run a bash shell in a container based in the build image.
 
+Our [changelog](./CHANGELOG.md) is powered by an automated [release-please](https://github.com/googleapis/release-please) action, which relies on commits following the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary).
+So make sure that **both your PRs and commmits follow the conventional commits format** for the CI validation to succeed.
+
+### CI
+
+The bulk of our CI work takes place in [Circle CI](https://app.circleci.com/pipelines/github/netlify/build-image).
+
+If **you're part of the Netlify org** and have write access to the repo our pipeline will take care of:
+- Linting the Dockerfile
+- Build your Dockerfile, tag it and push it to our [docker hub repo](https://hub.docker.com/r/netlify/build)
+- Run the automated [bats tests](#tests)
+
+
+If you **don't have access writes to the repo** and are submitting a PR via a forked repo, the CI pipeline will still execute for you. The main difference is that it won't push your built image to our docker repo.
+If required, someone with access writes to the repo can trigger the push for you, so if you require it (mainly useful for testing purposes) reach out to someone in the team :+1:
+
+## Releasing
+
+1. Merge the relevant package release PR created by `release-please`
+2. Wait for the CI pipelines to finish
+3. Create a PR in the [buildbot](https://github.com/netlify/buildbot) to bump [the version](https://github.com/netlify/buildbot/blob/0ada244ab84a1759a70d6b2cfc27c9987b5c77ca/.circleci/config.yml#L141-L150) of the `build-image`.
+
+### Running Test `buildbot` Releases
+
+If you want to test a particular `build-image` change before going through the regular release process you can do so by creating a PR following the process above :point-up: and pointing to any `build-image` you want. Any branch
```

Comment:
Is there an unintentional line break after "Any branch"?

Reply:
Not really I believe 🤔 but it does not impact the final document - https://github.com/netlify/build-image/blob/chore/update-contributing-ci/CONTRIBUTING.md